react/features/calendar-sync/components/MeetingList.native.js

```diff
@@ -60,6 +58,12 @@ class MeetingList extends Component<Props> {
     constructor(props) {
         super(props);
 
+        const { dispatch, displayed } = this.props;
```

Comment:
Not that it matters but you also have props passed into the constructor. They should be the same props, I think.

Reply:
True!